todo_app/oauth.py

```diff
@@ -0,0 +1,33 @@
+import os
+from flask import session
+from flask_dance.contrib.github import make_github_blueprint
+from flask_dance.consumer import oauth_authorized
+
+blueprint = make_github_blueprint(
+    client_id = os.getenv('OAUTH_CLIENT_ID'),
+    client_secret = os.getenv('OAUTH_CLIENT_SECRET'),
+    
+)
+def get_role_for_user(user_id):
+    # Should return 'writer' if user_id matches your GitHub id
+    # And 'reader' if not
+    user_id=session["username"]
+    #username=session["username"]
+    if user_id == os.getenv("OAUTHADMIN"):
+        role="ADMIN"
+    else:
+        role="READ ONLY" 
+    pass
```

Comment:
`pass` is a keyword in python explicitly designed to say "I'll finish writing this code later I swear!" and is fundamentally used as a placeholder - so let's get rid of it! But we probably do want this function to actually _return_ the role that we choose!
```suggestion
    return role
```